### ISSUE TITLE
Console: Add "API key settings" endpoints

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#6782]
Blocked by https://github.com/storj/storj/issues/6781

### AC:
* add an endpoint to the console server for creating an API key with settings at the same time
    * all values should come from the client side, except for "user ID", which should be automatically populated on the backend based on who is making the request
* add an endpoint to the console server for retrieving a list of API keys and settings
    * this endpoint should still return API key info if there is a row in api keys table, but not api key settings - in this case, just return empty settings
* add basic tests for these endpoints

**Comment by storjrobot** [comment]
This issue has been mentioned on **Storj Community Forum (official)**. There might be relevant details there:

https://forum.storj.io/t/backend-issues-to-work-on/27737/1

**Comment by VitaliiShpital** [comment]
closing in favor of https://github.com/storj/storj/issues/7400
